docs/_docs/collections.md

```diff
@@ -51,7 +51,7 @@ defaults:
 
 Create a corresponding folder (for example, `<source>/_my_collection`) and add
 documents. YAML Front Matter is read in as data if it exists, and everything
-after it is available in the document's `content` attribute. If no YAML Front
+after it is accessible via the document's `content` attribute. If no YAML Front
```

Comment:
putting a comma between a noun ("everything") and a verb ("is") would make this grammatically incorrect. https://www.grammarly.com/handbook/punctuation/comma/4/comma-with-subjects-and-verbs/